client/my-sites/plan-features/index.jsx

```diff
@@ -719,7 +721,8 @@ export default connect(
 				const currentPlan = sitePlan && sitePlan.product_slug;
 
 				// Show price divided by 12? Only for non JP plans, or if plan is only available yearly.
-				const showMonthlyPrice = ! isJetpack || ( ! relatedMonthlyPlan && showMonthly );
+				const showMonthlyPrice = ! isJetpack || isSiteAT || ( ! relatedMonthlyPlan && showMonthly );
+
```

Comment:
regression? (isSiteAT=true + relatedMonthlyPlan=true + showMonthly=true)

Reply:
I'm not happy about this condition - this is mostly a stop gap for the purpose of testing the monthly business plan.

**Here's how I read it:**
- `relatedMonthlyPlan` = we have found the same plan as the current one, with a monthly schedule
- `showMonthly` = current plan is not a monthly plan.

_Show prices divided by 12 if:_
**Currently:**
- If a site is not Jetpack
- If a site is Jetpack,  the current plan is not a monthly plan, and there is no equivalent monthly plan.

**My change:**
- If a site is not Jetpack or it is Atomic
- If a site is Jetpack and not atomic,  the current plan is not a monthly plan, and there is no equivalent monthly plan.